python-modules.sh

```diff
@@ -47,15 +60,16 @@ if [[ $FREETYPE_ROOT ]]; then
 fi
 perl -p -i -e "s|'darwin': \['/usr/local/'|'darwin': ['$INSTALLROOT'|g" setupext.py
 
-mkdir -p $INSTALLROOT/lib64/python2.7/site-packages $INSTALLROOT/lib/python2.7/site-packages
+mkdir -p $INSTALLROOT/lib/python$PYVER/site-packages \
```

Comment:
You were against it some time ago because you said it makes harder to do a "find" in the recipes :-)